src/ast.rs

```diff
@@ -22,14 +22,14 @@ pub enum LogicalExpression {
 }
 
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
This is a very small data structure, can it be `Copy` as well?

Reply:
fixed.